src/pages/matching.tsx

```diff
@@ -20,6 +20,7 @@ export interface BasicService {
     lineName: string;
     nocCode: string;
     operatorShortName: string;
+    description: string;
```

Comment:
Just having a poke around and it looks like the matching.ts API has an interface which might also need changing

Reply:
'serviceDescription now added as a key to MatchingData.